config/trustedproxy.php

```diff
@@ -1,27 +1,50 @@
 <?php
 
-// Pre-process any comma-separated values into an array.
-$trustedProxies = env('TRUSTED_PROXY_IP_ADDRESSES');
-if (str_contains($trustedProxies, ',')) {
-    $trustedProxies = explode(',', $trustedProxies);
-}
-
 return [
+
     /*
      * Set trusted proxy IP addresses.
      *
-     * Both IPv4 and IPv6 addresses are supported, along with CIDR notation.
+     * Both IPv4 and IPv6 addresses are
+     * supported, along with CIDR notation.
      *
-     * The "*" character is syntactic sugar within TrustedProxy to trust any proxy
-     * that connects directly to your server,a requirement when you cannot know the
-     * address of your proxy (e.g. if using Rackspace balancers).
+     * The "*" character is syntactic sugar
+     * within TrustedProxy to trust any proxy
+     * that connects directly to your server,
+     * a requirement when you cannot know the address
+     * of your proxy (e.g. if using ELB or similar).
      *
-     * The "**" character is syntactic sugar within TrustedProxy to trust not just any
-     * proxy that connects directly to your server, but also proxies that connect to
-     * those proxies, and all the way back until you reach the original source IP.
+     */
+    'proxies' => '*', // [<ip addresses>,], '*', '<ip addresses>,'
```

Comment:
According to the [`TrustedProxies` Class](https://github.com/fideloper/TrustedProxy/blob/9beebf48a1c344ed67c1d36bb1b8709db7c3c1a8/src/TrustProxies.php#L71) in the package, the `**` value is deprecated in favor of `*`. I checked and we use an env variable on each Rogue env on Heroku and specify it as `**`. By defaulting the config to `*` it should cover the same things and we can remove the env variables from each env on Heroku.